llvm/cmake/config-ix.cmake

```diff
@@ -52,6 +56,7 @@ else()
   check_include_file(mach/mach.h HAVE_MACH_MACH_H)
   check_include_file(malloc/malloc.h HAVE_MALLOC_MALLOC_H)
   check_include_file(pthread.h HAVE_PTHREAD_H)
+  check_include_file(sys/resource.h HAVE_SYS_RESOURCE_H)
```

Comment:
Do we need this? We already unconditionally include `sys/resource.h` from llvm/Support/Unix/Program.inc. IMO it would be better to do `#ifndef _WIN32` than the config check, which slows down configuration and complicates the build system, and has to be ported over to downstream build systems like gn and Bazel.

Reply:
I couldn't find good info on if `sys/resource.h` is present everywhere we care about, but if we already use it on unix systems then I'm fine removing the config checks.